### PR TITLE
Create New Sample - tab-support-offline - Nodejs

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/tab-support-offline/nodejs/README.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+## Deploying
+
+1. Build client `npm run build` 
+2. Build command aboves moves `client/build`  folder to server directory
+3. Deploy to Azure App Service
+
+## Further reading
+
+- [Tab](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/what-are-tabs?tabs=personal)
+
```
Comment (Pawank-MSFT) [inline samples/tab-support-offline/nodejs/README.md:176]: add azure blob storage also

**Review by Pawank-MSFT — COMMENTED** [review]
some comments please fix

File: README.md [hunk 2]
```diff
@@ -82,6 +82,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |27| Tab DeepLink Pass Values |Microsoft Teams sample which displays deeplinks based on the values passed dynamically to the context. ||[View][tab-deeplink-pass-values#ts]|
 |28| Tab Meeting Recording and transcript with auto recording |This sample shows meeting recording and transcript with auto recording. |[View][tab-meeting-auto-recording#csharp]||
 |29| Nested App Authentication |This sample shows nested app authentication (NAA) is a new protocol that simplifies single sign-on (SSO) for single page applications. ||[View][tab-nested-auth#nodejs] ![toolkit-icon](assets/toolkit-icon.png)|
+|29| Offline Support Tickets |A simple react application for Microsoft Teams. This app allows user to login using SSO and log attendence. ||[View][tab-support-offline#nodejs] ![toolkit-icon](assets/toolkit-icon.png)|[View](/samples/tab-support-offline/nodejs/demo-manifest/tab-support-offline.zip)
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply README.md:85]: Fixed

File: samples/tab-support-offline/nodejs/README.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
+---
+page_type: sample
+description: A simple react application for Microsoft Teams. This app allows user to login using SSO and log attendence.
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-support-offline/nodejs/README.md:3]: Fixed

File: samples/tab-support-offline/nodejs/README.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+extensions:
+ contentType: samples
+ createdDate: "08-28-2024 10:00:01"
+urlFragment: officedev-microsoft-teams-samples-tab-support-offline-nodejs
+
+---
+
+## Offline Support Tickets
+
+A classic CRUD app working in offline mode inside Microsoft Teams.
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-support-offline/nodejs/README.md:19]: Fixed

File: samples/tab-support-offline/nodejs/README.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+## Try it yourself - experience the App in your Microsoft Teams client
+Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app package (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).
+
+**Microsoft Teams offline support tickets sample app:** [Manifest](/samples/tab-support-offline/nodejs/demo-manifest/tab-support-offline.zip)
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/)
+- [dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) or [ngrok](https://ngrok.com/) latest version or equivalent tunnelling solution
+- [VS Code](https://code.visualstudio.com/)
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-support-offline/nodejs/README.md:34]: Fixed

File: samples/tab-support-offline/nodejs/README.md (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-support-offline/nodejs/README.md:176]: Fixed

File: samples/tab-support-offline/nodejs/README.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+    ```
+- The client will start running on 3000 port
+
+4. Open blobStoreOperations.js file from this path folder `samples\tab-support-offline\nodejs\server\` and update:
+   - `{{ account-Name }}` - Replace these values with your Azure Storage account details
+   - `{{ container-Name }}`
+   
+**This step is specific to Teams:**
+
+- **Edit** the `manifest.json` contained in the  `appManifest` folder to replace your Microsoft App Id `{{YOUR-MICROSOFT-APP-ID}}` (that was created when you registered your bot earlier) *everywhere* you see the place holder string `{{YOUR-MICROSOFT-APP-ID}}` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-support-offline/nodejs/README.md:117]: Fixed

File: samples/tab-support-offline/nodejs/README.md (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-support-offline/nodejs/README.md:176]: Fixed

File: samples/tab-support-offline/nodejs/README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-support-offline/nodejs/README.md:117]: Fixed

File: samples/tab-support-offline/nodejs/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/tab-support-offline/nodejs/README.md:34]: Fixed

**Review by Pawank-MSFT — APPROVED** [review]
Please resolve the comment, Approving!

**Review by Mohammed-MSFT — APPROVED** [review]
Changes looks fine, approving

**Pull request merged**